Hush/Assets/Scripts/Enemies/Enemy.cs

```diff
@@ -354,6 +378,13 @@ private void SetState(EnemyState state)
                 GameManager.Instance.EnemyStoppedAttacking(ID);
             }
         }
+        
+        private IEnumerator Reveal()
+        {
+            Show();
+            yield return new WaitForSeconds(1.0f);
```

Comment:
Would be cool if this was a serializeField

Reply:
Changed in acb305f